driver.md

```diff
@@ -266,8 +266,26 @@ Note that `TEST` is sorted here so that key `"network"` comes before key `"pre"`
     rule <k> run TESTID : { KEY : (VAL:JSON) , NEXT , REST } => run TESTID : { NEXT , KEY : VAL , REST } ... </k>
       requires KEY in #execKeys
 
-    rule <k> run TESTID : { "exec" : (EXEC:JSON) } => load "exec" : EXEC ~> start ~> flush ... </k>
+    rule <k> run TESTID : { "exec" : (EXEC:JSON) } => loadCallState EXEC ~> start ~> flush ... </k>
     rule <k> run TESTID : { "lastblockhash" : (HASH:String) } => #startBlock ~> startTx    ... </k>
+
+    rule <k> load "exec" : J => loadCallState J ... </k>
```

Comment:
Seems that this rule and the rule above for `run TESTID : { "exec" : EXEC } => loadCallState EXEC ~> start ~> flush ... </k>` are redundant. Is this one needed?

Reply:
I don't know if the rule above is needed. But the rule I added is needed for some `.evm` test scripts.

Reply:
Can we translate them to `loadCallState` instead?

Reply:
It didn't work right away.